apps/frontend/src/libs/components/button/button.tsx

```diff
@@ -3,16 +3,20 @@ import { getValidClassNames } from "~/libs/helpers/helpers.js";
 import styles from "./styles.module.css";
 
 type Properties = {
+	children?: React.ReactNode;
 	isFluid?: boolean;
+	isIconButton?: boolean;
 	isPrimary?: boolean;
-	label: string;
+	label?: string;
```

Comment:
```suggestion
	label: string;
```
Label is mandatory. Add `hasVisuallyHiddenLabel` prop and if it is true, use `visually-hidden` class to hide label